fedora_elections/elections.py

```diff
@@ -233,20 +233,12 @@ def vote_simple(election):
 
     if form.validate_on_submit():
         if form.action.data == 'submit':
-            for candidate in form:
-                if candidate.short_name in ['csrf_token', 'action']:
-                    continue
-
-                new_vote = models.Vote(
-                    election_id=election.id,
-                    voter=flask.g.fas_user.username,
-                    timestamp=datetime.now(),
-                    candidate_id=candidate.data,
-                    value=1,
-                )
-                SESSION.add(new_vote)
-            SESSION.commit()
-
+            candidates =  [
+                candidate
+                for candidate in form
+                if candidate and candidate.short_name not in ['csrf_token', 'action']
+            ]
+            process_vote(candidates, election, votes, revote, None, 1)
```

Comment:
ah, never knew you can do that :) I am learning all sorts of things today, thanks!